src/pilot/deployment-validation/sanity_test.sh

```diff
@@ -642,6 +643,80 @@ setup_cinder(){
   done
 }
 
+setup_manila(){
+  info "### Manila test"""
+  
+  set_admin_scope
+  manila_exists=$(manila service-list | grep manila-share |  head -n 1  | awk '{print $4}')
+  echo "$manila_exists";
```

Comment:
No need for this echo line

Reply:
Will remove. I guess I had these for debugging